lua/telescope/builtin/init.lua

```diff
@@ -11,13 +11,17 @@
 ---
 --- :lua require('telescope.builtin').$NAME_OF_PICKER()
 ---
---- To use any of Telescope's default options or any picker-specific options, call your desired picker by passing a lua
---- table to the picker with all of the options you want to use. Here's an example with the live_grep picker:
+--- To configure your desired picker, call it by passing a lua table with all
+--- of the options you want to use. Pickers have their own specific set of
+--- options, but also accept a subset of the defaults options. The default
+--- options which are valid to configure a picker are marked as
+--- "Picker-Overridable" in telescope.setup.defaults
```

Comment:
It'll probably be easier to mark options that aren't "picker-overriable". 
In theory, all but a small handful can at least in technically be overriden by the each individual picker.

I would reword this bit slightly and let's leave out the marking of individual options an "Picker-Overridable" for now.